src/stream/src/executor/managed_state/aggregation/extreme.rs

```diff
@@ -81,6 +82,10 @@ where
 
     /// The sort key serializer
     serializer: ExtremeSerializer<A::OwnedItem, EXTREME_TYPE>,
+
+    /// Consistent hash value to be set in value meta. Used for grouping the kv together in
+    /// storage. Each state will have the same consistent hash value.
+    consistent_hash_value: u16,
```

Comment:
We may propose some shorter name for `consistent_hash_value` 🤔

Reply:
Indeed. What about `place_key`?